### PR TITLE
Feature/#044 testinicio

### PR DESCRIPTION
**Pull request by ezegonmac**
Add test for home view.
Test title, podium and events.
Change tests so Github Actions can execute them.

Now to run the view tests from the console with the command 'python manage.py test' it is necessary to download the chrome driver (https://chromedriver.chromium.org/downloads) and add the folder in which it is located to the environment variables.

### CODE REVIEW

**Review by TomasCB24 — APPROVED** [review]
All is okey.

File: innoweb/tests.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
```
Comment (IsmaelP19) [inline innoweb/tests.py:5]: Lines 4 and 5 should be deleted as they are imports unused.

**Review by IsmaelP19 — APPROVED** [review]
All tests running OK. There are some lines which should be deleted since they are not used.
Good job!! 🚀🙌

File: innoweb/tests.py (hunk 1 above)
Comment (ezegonmac) [reply innoweb/tests.py]: OK, fixed 👌🏻

**Pull request merged**